docs/client.rst

```diff
@@ -543,23 +546,30 @@ If there is an error, a status error will be sent by the agent.
 
 Write access profile
 ````````````````````
-These functions are generated automatically by `MicroRTPSGen` utility with the ``-write-access-profile`` option enabled over an idl file with a topic `TOPICTYPE`.
-The declaration of these function can be found in the generated file ``TOPICTYPEWriter.h``.
+These functions are enabled when PROFILE_WRITE_ACCESS is enabled into ``client.config`` file.
```

Comment:
Use "in" instead of "into".
"Enabled in the ..."

Reply:
Done!